src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Circle extends Figure {
+    private double radius;
+    private double area;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+        this.area = calculateArea();
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead of directly calculating the area in the constructor, consider calling 'calculateArea()' in the 'getArea()' method when it's actually needed.